packages/models-library/src/models_library/api_schemas_resource_usage_tracker/licensed_items_purchases.py

```diff
@@ -0,0 +1,56 @@
+from datetime import datetime
+from decimal import Decimal
+from typing import NamedTuple
+
+from models_library.licensed_items import LicensedItemID
+from models_library.products import ProductName
+from models_library.resource_tracker import PricingUnitCostId
+from models_library.resource_tracker_licensed_items_purchases import (
+    LicensedItemPurchaseID,
+)
+from models_library.users import UserID
+from models_library.wallets import WalletID
+from pydantic import BaseModel, ConfigDict, PositiveInt
+
+
+class LicensedItemPurchaseGet(BaseModel):
+    licensed_item_purchase_id: LicensedItemPurchaseID
```

Comment:
TIP:  i think i simple id is enough here but you like verbosity :-)

Reply:
yes ;)